src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCuiCallbackHandler.java

```diff
@@ -55,10 +57,16 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse aboutToSubmit(CallbackParams callbackParams) {
```

Comment:
my suggestion in this part is:
    private CallbackResponse aboutToSubmit(CallbackParams callbackParams) {
        CaseData updatedData = getUpdatedCaseData(callbackParams);

        boolean responseLanguageIsBilingual = respondentLiPResponse.isRespondentResponseBilingual(updatedData);
        AboutToStartOrSubmitCallbackResponse.AboutToStartOrSubmitCallbackResponseBuilder responseBuilder =
            AboutToStartOrSubmitCallbackResponse.builder().data(updatedData.toMap(objectMapper));

        if (!responseLanguageIsBilingual) {
            responseBuilder.state(CaseState.AWAITING_APPLICANT_INTENTION.name());
        }

        return responseBuilder.build();
    }